### PR TITLE
TLS etc doc amendments.

### PR DESCRIPTION
**Pull request by jimis**

### CODE REVIEW

**Comment by jimis** [comment]
@nickanderson please have a look.

**Comment by jimis** [comment]
Merging in order to see changes on docs master. TODO cherry-pick to 3.7, and cherry-pick the secure bootstrap guide to 3.6.

File: reference/components/cf-serverd.markdown [hunk 1]
```diff
@@ -141,16 +145,26 @@ For a list of possible ciphers, see man page for "openssl ciphers".
...
 
-**See also:** [`protocol_version`][Components and Common Control#protocol_version], [`tls_ciphers`][Components and Common Control#tls_ciphers], [`tls_min_version`][Components and Common Control#tls_min_version], [`allowtlsversion`][cf-serverd#allowtlsversion], [`encrypt`][files#encrypt], [`logencryptedtransfers`][cf-serverd#logencryptedtransfers], [`ifencrypted`][access#ifencrypted]
+**See also:**
+[`protocol_version`][Components and Common Control#protocol_version],
+[`tls_ciphers`][Components and Common Control#tls_ciphers],
+[`tls_min_version`][Components and Common Control#tls_min_version],
+[`allowtlsversion`][cf-serverd#allowtlsversion],
+[`encrypt`][files#encrypt],
+[`logencryptedtransfers`][cf-serverd#logencryptedtransfers],
+[`ifencrypted`][access#ifencrypted]
```
Comment (nickanderson) [inline reference/components/cf-serverd.markdown]: nice, i like this formatting better in the markdown.

**Comment by nickanderson** [comment]
I am not in love with removing the old docs for deprecated things. As long as they are valid to the parser, they should be listed in the docs but noted as deprecated. Otherwise things look prety good.

**Comment by jimis** [comment]
@nickanderson what do you think on pull request #1182 then? Simply mentioning the attributes, if somebody wants to figure them out he can look old versions of the manual.

**Comment by nickanderson** [comment]
Thats a fair compromise.

**Comment by nickanderson** [comment]
Are you going to do the same for the rest of the deprecated attributes in the docs?

**Comment by jimis** [comment]
On Wed, Sep 23, 2015 at 9:49 PM, Nick Anderson <notifications@github.com>
wrote:

> Are you going to do the same for the rest of the deprecated attributes in
> the docs?
>

I think it would be nice, but no, I don't have time to go through them all.

**Pull request merged**